authorization/2.0/_index.md

```diff
@@ -10,17 +10,17 @@ these permissions.
 
 * [Jakarta Authorization 2.0 Release Record]()
```

Comment:
It would be great if you could fill in the url for your release record, but I won't hold up the merge for it.  I'll approve without it and merge tomorrow.  Thanks.
https://projects.eclipse.org/projects/ee4j.jacc/releases/2.0.0